release-notes/unreleased/LUN-366-progress-indicator-line-separated.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: LUN-366
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Level-Fortschrittslinie leicht verlängert, um Lücken zu vermeiden
```

Comment:
It is? Just wondering because in the IGAPP release notes it says DE is required for the stores, and in LUN the `createReleaseNotes.sh` script only creates the de tag.